lzy/execution-env/src/main/java/ai/lzy/env/aux/CondaPackageRegistry.java

```diff
@@ -14,167 +19,287 @@ public class CondaPackageRegistry {
     private static final Logger LOG = LogManager.getLogger(CondaPackageRegistry.class);
     private static final String VERSION_REGEX = "(?=>=)|(?=<=)|(?=<)|(?=>)|==|=";
     private static final int SPLIT_LIMIT = 2;
+    private static final String DEFAULT_ENV_NAME = "base";
+    private static final String DEFAULT_PYTHON_VERSION = "3.10";
+    private static final String DEFAULT_PYPI_INDEX = "https://pypi.org/simple";
 
-    private final Map<String, Map<String, Package>> envs = new HashMap<>();
+    // TODO(artolord) remove this ugly hack after removing conda.yaml
+    private static final Map<String, String> NAME_TO_PYTHON_VERSION = Map.of(
+        "py37", "3.7",
```

Comment:
let's drop it

Reply:
I will remove this after replacing conda.yaml with a few fields in proto. This will be in the next pull request.